irc/handlers.go

```diff
@@ -1741,6 +1741,13 @@ func cmodeHandler(server *Server, client *Client, msg ircmsg.Message, rb *Respon
 			return false
 		}
 	}
+
+	if msg.Command == "SAMODE" {
```

Comment:
Thanks!

Nit: can you factor this test out into a single `isSamode := msg.Command == "SAMODE"`, since it's also used below?

Reply:
Done.